openmetadata-service/src/main/java/org/openmetadata/service/resources/dqtests/TestCaseResolutionStatusResource.java

```diff
@@ -0,0 +1,292 @@
+                @Content(
+                    mediaType = "application/json",
+                    schema = @Schema(implementation = TestCaseResolutionStatusResultList.class)))
+      })
+  public ResultList<TestCaseResolutionStatus> list(
+      @Context SecurityContext securityContext,
+      @Parameter(description = "Test Case ID", schema = @Schema(type = "UUID")) @QueryParam("testCaseId")
+          UUID testCaseId,
+      @Parameter(description = "Limit the number tests returned. (1 to 1000000, default = 10)")
+          @DefaultValue("10")
+          @QueryParam("limit")
+          @Min(0)
+          @Max(1000000)
+          int limitParam,
+      @Parameter(description = "Returns list of tests at the offset", schema = @Schema(type = "string"))
+          @QueryParam("offset")
+          String offset,
+      @Parameter(
+              description = "Filter test case statuses after the given start timestamp",
+              schema = @Schema(type = "number"))
+          @NonNull
+          @QueryParam("startTs")
+          Long startTs,
+      @Parameter(
+              description = "Filter test case statuses before the given end timestamp",
+              schema = @Schema(type = "number"))
+          @NonNull
+          @QueryParam("endTs")
+          Long endTs,
+      @Parameter(
+              description = "Filter test case statuses by status",
+              schema = @Schema(implementation = TestCaseResolutionStatusTypes.class))
+          @QueryParam("testCaseResolutionStatusType")
+          String testCaseResolutionStatusType,
+      @Parameter(description = "Only list the latest statuses", schema = @Schema(type = "Boolean"))
+          @DefaultValue("false")
+          @QueryParam("latest")
+          Boolean latest,
+      @Parameter(description = "Filter test case statuses by assignee", schema = @Schema(type = "String"))
+          @QueryParam("assignee")
+          String assignee,
+      @Parameter(description = "Filter test case statuses by reviewer", schema = @Schema(type = "String"))
+          @QueryParam("reviewer")
+          String reviewer,
+      @Parameter(description = "Test case fully qualified name", schema = @Schema(type = "String"))
+          @QueryParam("testCaseFQN")
+          String testCaseFQN) {
+    OperationContext operationContext = new OperationContext(Entity.TEST_CASE, MetadataOperation.VIEW_ALL);
+    ResourceContextInterface resourceContext = ReportDataContext.builder().build();
+    authorizer.authorize(securityContext, operationContext, resourceContext);
+
+    ListFilter filter = new ListFilter(null);
+    filter.addQueryParam("testCaseResolutionStatusType", testCaseResolutionStatusType);
+    filter.addQueryParam("assignee", assignee);
+    filter.addQueryParam("reviewer", reviewer);
+    filter.addQueryParam("entityFQNHash", testCaseFQN);
+
+    return repository.list(offset, startTs, endTs, limitParam, filter, latest);
+  }
+
+  @GET
+  @Path("/sequenceId/{sequenceId}")
```

Comment:
Can we use id or stateId( as in the state machine step) . SequenceID has a different understanding of it.